### ISSUE TITLE
 **What steps will reproduce the problem?**

### ISSUE THREAD

**Issue by PG408** [spyder-ide/spyder#4935]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

KeyError: None

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 3.6.1
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.1 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.5.6 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.1.1 (OK)
sympy >=0.7.3    :  1.0 (OK)
cython >=0.21    :  0.25.2 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  5.3.0 (OK)
pylint >=0.25    :  1.6.4 (OK)

```

**Comment by ccordoba12** [comment]
What were you doing when you got this error?

**Comment by PG408** [comment]
I don't knw, I  just use IPython to run my code, Plotting something, then the error window came up.

**Comment by PG408** [comment]
This came again, 
when I push "Analyze" button in "static code analysis"  windows
![image](https://user-images.githubusercontent.com/20208650/29203532-2efa832e-7ea4-11e7-9b44-2b6b5e575e7e.png)

**Comment by ccordoba12** [comment]
This is a duplicate of issue #4922 and it'll be fixed in Spyder 3.2.1

**Comment by PG408** [comment]
g
